azure-templates/helm-template.yml

```diff
@@ -0,0 +1,36 @@
+
+parameters:
+  - name: 'aksConnection'
+    type: string
+    default: 'k8s-team3-config-dev'
+  - name: 'releaseName'
+    type: string
+  - name: 'nameSpace'
+    type: string
+  - name: 'chartName'
+    type: string
+  - name: 'overrides'
+    type: string
+    default: ''
+
+steps:
+  - task: HelmDeploy@0
+    displayName: Package Helm Chart
+    inputs:
+      command: 'package'
+      chartPath: './${{ parameters.chartName }}/'
+      destination: './helm/'
+  
+  - task: HelmDeploy@0
```

Comment:
ideally the deploy would be in a deployment job recorded against an environment, and the chart archive could be an artifact.

Reply:
Moved to a deployment job and the chart package is being uploaded as an artifact.